.github/workflows/ci.yml

```diff
@@ -11,8 +11,12 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v4
-      - uses: ./.github/composite-actions/setup-node
-      - run: pnpm lint
+      - name: Setup Biome
+        uses: biomejs/setup-biome@v2
+        with:
+          version: latest
+      - name: Run Biome
+        run: biome ci .
```

Comment:
It took 6 seconds to complete! Amazing!

https://github.com/route06/urql-exhaustive-additional-typenames-exchange/actions/runs/8281901363/job/22661476104?pr=4